generator/build.gradle.kts

```diff
@@ -16,9 +16,9 @@ buildscript {
 
 plugins {
     base
-    kotlin("jvm") version "1.5.21" apply false
-    id("org.jetbrains.dokka") version "1.5.0" apply false
-    id("com.diffplug.spotless") version "5.14.2" apply false
+    kotlin("jvm") version "1.6.21" apply false
```

Comment:
Curious why 1.6 over 1.7?

Reply:
good question, think I tried going straight to 1.7 and ran into errors relating to build plugins and kotlin-test configuration.   Circled back after your comment and seem to have it working, so will push the 1.7 upgrade on this PR.